Sources/swift-parsing-benchmark/StringAbstractions.swift

```diff
@@ -23,15 +23,15 @@ let stringAbstractionsSuite = BenchmarkSuite(name: "String Abstractions") { suit
 
   suite.benchmark("Substring") {
     var input = input[...].utf8
-    let output = Many(Int.parser(), separator: StartsWith("é").utf8).parse(&input)
+    let output = Many(Int.parser(), separator: "é".utf8).parse(&input)
```

Comment:
We had some tests still using `StartsWith` that I noticed, so did a lil unrelated cleanup.